### PR TITLE
Enzyme to RTL (localization, execute)

### PR DESCRIPTION
**Pull request by NestorEncinas**
- Migrate localization and execute tests to RTL

JIRA: FET-1054

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by NestorEncinas** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9190/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11019/ : FAILURE in 17m 40s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9191/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1784/ : UNSTABLE in 38m 19s

**Comment by NestorEncinas** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9198/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11025/ : SUCCESS in 16m 34s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1786/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9199/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1786/ : SUCCESS in 33m 34s

File: libs/sdk-ui/src/base/tests/testHelper.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// (C) 2022 GoodData Corporation
+import { render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+export const setupComponent = (jsx: any) => {
```
Comment (xMort) [inline libs/sdk-ui/src/base/tests/testHelper.tsx:5]: Isn't there some type that can satisfy it?

File: libs/sdk-ui/src/execution/tests/Execute.test.tsx [hunk 2]
```diff
@@ -189,9 +190,9 @@ describe("Execute", () => {
 
         await createDummyPromise({ delay: 100 });
 
-        dummyExecutor.update();
+        // dummyExecutor.update();
```
Comment (xMort) [inline libs/sdk-ui/src/execution/tests/Execute.test.tsx:193]: Forgotten comment.

Comment (NestorEncinas) [reply libs/sdk-ui/src/execution/tests/Execute.test.tsx]: Removed.

File: libs/sdk-ui/src/base/tests/testHelper.tsx (hunk 1 above)
Comment (NestorEncinas) [reply libs/sdk-ui/src/base/tests/testHelper.tsx:5]: Added it.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9210/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11034/ : SUCCESS in 16m 03s

**Review by xMort — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2835

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Pull request merged**